docker-compose.yml

```diff
@@ -1106,6 +1106,20 @@ services:
       - EX_APPS_NET=${EX_APPS_NET:-ipv4@localhost}
       - EX_APPS_COUNT=${EX_APPS_COUNT:-50}
 
+  appapi-dsp-harp:
+    # todo
+    image: nextcloud-appapi-harp:latest
+    container_name: nextcloud-appapi-harp
+    network_mode: ${NETWORK_MODE:-master_default}
+    ports:
+      - "${EXAPPS_HTTP_PORT:-8780}:8780"
+      - "${EXAPPS_FRP_PORT:-8782}:8782"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    environment:
+      - NC_HARP_SHARED_KEY=${NC_HAPROXY_SHARED_KEY:-some_very_secure_password}
+      - NC_INSTANCE_URL=${NC_INSTANCE_URL:-http://nextcloud.local}
+      - HP_FRP_DISABLE_TLS=${HP_FRP_DISABLE_TLS:-true}
```

Comment:
do not needed, better to check without this flag

Reply:
those value is related to the communication type between `HaRP` and `ExApp` only.
we will describe this in the documentation when will work on it.

by default we will always recomend to **not disable** it.